src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -235,6 +246,8 @@ private void MethodChecks(MethodDeclarationSyntax syntax, Binder withTypeParamsB
                 else if (!IsStatic)
                 {
                     diagnostics.Add(ErrorCode.ERR_BadExtensionMeth, location);
+                    // PROTOTYPE(readonly-ref): to match PEMethodSymbol, should we SetIsExtensionMethod(false)? See #20282 for more details
```

Comment:
Agreed. I'll replace with a CONSIDER and a link to the bug.

---
In reply to: [122884419](https://github.com/dotnet/roslyn/pull/19703#discussion_r122884419) [](ancestors = 122884419)